flask_rebar/request_utils.py

```diff
@@ -52,22 +52,19 @@ def __getitem__(self, key):
         return self.headers[key]
 
 
-def get_json_from_resp(resp):
-    return json.loads(resp.data.decode("utf-8"))
-
-
 def response(data, status_code=200, headers=None):
     """
     Constructs a flask.jsonify response.
 
     :param dict data: The JSON body of the response
     :param int status_code: HTTP status code to use in the response
+    :param dict headers: Additional headers to attach to the response
     :rtype: flask.Response
     """
-    resp = jsonify(data)
-
-    if not get_json_from_resp(resp=resp):
-        resp.data = ""
+    if data is None:
+        resp = Response(mimetype="application/json")
```

Comment:
I don't love specifying this here, but if we don't, then we get `text/plain` from [werkzeug](https://github.com/pallets/werkzeug/blob/0.15.2/src/werkzeug/wrappers/base_response.py#L139), and jsonify barfs on passing None as args on old versions of [Flask](https://github.com/pallets/flask/blob/0.10/flask/json.py#L222).

Reply:
worth pulling out this string literal into a meaningfully-named constant (`DEFAULT_MIMETYPE` or some such)?

Reply:
(sorry for the noise, realized I was looking at an outdated diff, please ignore. deleting the previous comment and this one now.)